backend/source/archive/models/google_takeout.py

```diff
@@ -16,13 +16,29 @@
 
 
 def e7_to_decimal(e7_coordinate: int) -> float:
-    return float(e7_coordinate) / 10000000
+    try:
+        return float(e7_coordinate) / 10000000
+    except:
```

Comment:
Be explicit about the exceptions you catch. It's both a form of documentation, and a way to avoid catching the wrong things